src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,88 @@
+import { Typography } from "@mui/material";
+import { fetchArticleById } from "../../api";
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid2";
+import { styled } from "@mui/material/styles";
+import Paper from "@mui/material/Paper";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import ThumbDownIcon from "@mui/icons-material/ThumbDown";
+import IconButton from "@mui/material/IconButton";
+
+function ArticlePage() {
+  const [articleTitle, setArticleTitle] = useState();
+  const [articleImg, setArticleImg] = useState();
+  const [articleAuthor, setArticleAuthor] = useState();
+  const [currentArticle, setCurrentArticle] = useState();
+  const [articleDate, setArticleDate] = useState();
+  const [voteCount, setvoteCount] = useState();
+  const { article_id } = useParams();
+
+  const Item = styled(Paper)(({ theme }) => ({
+    backgroundColor: "#fff",
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+    ...theme.applyStyles("dark", {
+      backgroundColor: "#1A2027",
+    }),
+  }));
+
+  useEffect(() => {
+    fetchArticleById(article_id).then((article) => {
+      setArticleTitle(article.title);
+      setCurrentArticle(article.body);
+      setArticleImg(article.article_img_url);
+      setArticleAuthor(article.author);
+      setArticleDate(article.created_at);
+      setvoteCount(article.votes);
+    });
```

Comment:
This is a lot of state to change - you could alternatively have a singleArticle state that is an object and then set the article you get back to that single piece of state. 

Then below you could do singleArticle.articleTitle etc. The more state changes you have the more stress you're putting on your app